res/themes/light/css/_light.scss

```diff
@@ -252,24 +268,24 @@ $e2e-warning-color: #ba6363;
 
 /*** ImageView ***/
 $lightbox-bg-color: #454545;
-$lightbox-fg-color: #ffffff;
-$lightbox-border-color: #ffffff;
+$lightbox-fg-color: $background;
```

Comment:
This doesn't feel right: I think we're trying to hardcode the image view styles, not make them variable.